### PR TITLE
add "search with github" to header

### PR DESCRIPTION
**Pull request by qpwo**

### CODE REVIEW

**Comment by qpwo** [comment]
Bump. Here's a demo video.

https://user-images.githubusercontent.com/10591373/139111778-25fefb3d-0fbe-4f75-9e46-46c9528b1327.mov

**Comment by qpwo** [comment]
Bump for this PR – I think it's really simple and adds a lot of value

File: index.html [hunk 1]
```diff
@@ -24,6 +24,10 @@
             <i id="line2"></i>
             <i id="line3"></i>
         </div>
+        <span class="searchbar-wrap">
+            <input type="text" placeholder="search github" id="searchbar-input"/>
+            <a href="" target="_blank" id="searchbar-a">🔎</a>
+        </span>
```
Comment (bigtimebuddy) [inline index.html]: Why don't you do this with a simple form:

```html
<form method="get" action="https://github.com/pixijs/examples/search">
  <input type="text" name="q" placeholder="Search GitHub...">
  <button type="submit">Search</button>
</form>
```

Then you don't need any JavaScript.

**Comment by qpwo** [comment]
You're right that is better. I made that change.

**Comment by bigtimebuddy** [comment]
Could you revert the changes on the built JS?

**Comment by qpwo** [comment]
Yeah I reverted the built js and in a separate commit the built css, so I think you can exclude that commit from the PR if you want to keep the built css.

**Review by bigtimebuddy — APPROVED** [review]

**Comment by bigtimebuddy** [comment]
Thanks for your help and addition here @qpwo

**Pull request merged**